dev.tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "include": ["packages/**/*", "typings/**/*"],
+  "compilerOptions": {
+    "module": "system",
+    "jsx": "react",
+    "allowJs": false,
+    "pretty": true,
+    "strictNullChecks": true,
+    "noUnusedLocals": true,
+    "strict": true,
+    "allowSyntheticDefaultImports": true,
+    "moduleResolution": "node",
+    "baseUrl": ".",
+    "outFile": "/dev/null",
+    "tsBuildInfoFile": "./tsconfig.tsbuildinfo",
+    "incremental": true
```

Comment:
we can't just use "noEmit" because it would also prevent creating the `.tsbuildinfo` file and slow down type-checking